docs/maintain/maintain-guides-democracy.md

```diff
@@ -283,6 +283,17 @@ through `democracy.votingOf` extrinsic.
 
 ## Undelegate a Vote
 
+:::caution
+
+Currently, the undelegation logic has a flaw in it. If you choose to delegate and undelegate two
```

Comment:
Technically, when you reduce your delegation, the whole of the initial amount is set to unlock. If then, after the unlock period, you remove the lock, only the difference will become transferable because there is the lock of the current delegation.

So, if you delegate 500 DOT and then change it to 200 DOT, the lock set to expire is for the 500 DOT. When that lock expires and you remove it, 300 DOT will become transferable, because the only lock in existence then will be the one for 200 DOT.

The issue is that if you do this (the simplest case):
1. Delegate 500 DOT
2. Delegate 200 DOT at time t -> This creates a lock for 200 DOT and sets the lock for 500 DOT to expire at t + unlocking period`
3. Undelegate at `t1 < t + unlocking period` (i.e. before the previous lock expired) -> Resets the lock for 500 DOT to expire at `t1 + unlocking period`

The expected behaviour would be that 500 DOT still unlocks at t + unbonding period and the 200 DOT at t1 + unbonding period.

Obviously, this only affects accounts that **delegated with a conviction**. 

Also, I wouldn't call this a "flaw". It's an edge case that wasn't considered.

Reply:
> The formatting for the caution is off. Check https://docusaurus.io/docs/markdown-features/admonitions for the correct way to use admonitions.
> 
> <img alt="image" width="910" src="https://user-images.githubusercontent.com/86818441/193341628-b3c49826-f5ca-40d6-8adf-b87ba36f0d92.png">

Ah, this seems to be an issue with prettier:
If the closing colons are on the next line, it puts it at the end of the sentence.